core/server/data/migration/fixtures/006/01-transform-dates-into-utc.js

```diff
@@ -0,0 +1,201 @@
+var config = require('../../../../config'),
+    models = require(config.paths.corePath + '/server/models'),
+    sequence = require(config.paths.corePath + '/server/utils/sequence'),
+    moment = require('moment'),
+    Promise = require('bluebird'),
+    messagePrefix = 'Transform into UTC Dates: ',
+    settingsKey = '006/01',
+    _private = {};
+
+_private.getTZOffset = function getTZOffset(date) {
+    return date.getTimezoneOffset();
+};
+
+_private.getTZOffsetMax = function () {
+    return Math.max(Math.abs(new Date('2015-07-01').getTimezoneOffset()), Math.abs(new Date('2015-01-01').getTimezoneOffset()));
+};
+
+_private.addOffset = function (date) {
+    if (_private.noOffset) {
+        return moment(date).toDate();
+    }
+
+    return moment(date).add(_private.getTZOffset(date), 'minutes').toDate();
+};
+
+module.exports = function transformDatesIntoUTC(options, logger) {
+    var ServerTimezoneOffset = _private.getTZOffsetMax(),
+        settingsMigrations = null;
+
+    options.migration = true;
+
+    return sequence([
+        function () {
```

Comment:
I would prefer to give these functions names please? It would help in certain cases with stack traces & also make the code a little bit easier to read.